ghost/core/core/frontend/public/tracker.js

```
@@ -0,0 +1,289 @@
+    let _payload = JSON.stringify(payload)
+    attributesToMask.forEach(attr => {
+      _payload = _payload.replaceAll(
+        new RegExp(`("${attr}"):(".+?"|\\d+)`, 'mgi'),
+        '$1:"********"'
+      )
+    })
+
+    return _payload
+  }
+
+  /**
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(indent)